decidim-admin/spec/commands/decidim/admin/create_participatory_space_private_user_spec.rb

```diff
@@ -74,6 +76,32 @@ module Decidim::Admin
 
           subject.call
         end
+
+        context "when the current users are not to be deleted" do
+          let(:user1) { create(:user, organization: privatable_to.organization) }
+
+          before do
+            Decidim::ParticipatorySpacePrivateUser.create!(decidim_user_id: user1.id, privatable_to_id: privatable_to.id, privatable_to_type: privatable_to.class.to_s)
+          end
+
+          it "doesn't suppress the existing users" do
+            expected_number = Decidim::ParticipatorySpacePrivateUser.where(privatable_to_id: privatable_to.id, privatable_to_type: privatable_to.class.to_s).count
+
+            subject.call
+
+            expect(Decidim::ParticipatorySpacePrivateUser.where(privatable_to_id: privatable_to.id, privatable_to_type: privatable_to.class.to_s).count).to equal(1 + expected_number)
```

Comment:
I think this can be a little more readable by using the scope and/or the factory. Also it'd be a good place to use the change rspec matcher, something like `expect { subject.call }.to change(xxx, :count).by(1) `